soql-analyzer/src/test/scala/com/socrata/soql/analyzer2/rollup/RollupExactTest.scala

```diff
@@ -379,4 +379,28 @@ class RollupExactTest extends FunSuite with MustMatchers with RollupTestHelper w
       TestRollupExact(select, rollup, analysis.labelProvider) must be (None)
     }
   }
+
+  test("parameterless aggregate one-to-one rewrite bug") {
```

Comment:
Nit: I think it would helpful as a reader of this code if the spec description actually called out desired behavior rather than referring to the bug.

Something like this?
```scala
test("rollup on an an aggregate with a parameter cannot be applied to a query with the same groupby clause but a different aggregate and parameter")
```

Reply:
That's not quite the bug though; it was specifically the fact that "count(*)" has no parameters that was the trigger.  It is indeed not an ideal name though.